SMLHelper/Options/ModOptions.cs

```diff
@@ -72,9 +73,19 @@ public ModOptions(string name)
         /// <para>This method should be composed of calls into the following methods: 
         /// <seealso cref="AddSliderOption"/> | <seealso cref="AddToggleOption"/> | <seealso cref="AddChoiceOption(string, string, string[], int)"/> | <seealso cref="AddKeybindOption(string, string, GameInput.Device, KeyCode)"/>.</para>
         /// <para>Make sure you have subscribed to the events in the constructor to handle what happens when the value is changed:
-        /// <seealso cref="SliderChanged"/> | <seealso cref="ToggleChanged"/> | <seealso cref="ChoiceChanged"/> | <seealso cref="KeybindChanged"/> | <seealso cref="DropdownChanged"/>.</para>
+        /// <seealso cref="SliderChanged"/> | <seealso cref="ToggleChanged"/> | <seealso cref="ChoiceChanged"/> | <seealso cref="KeybindChanged"/>.</para>
         /// </summary>
         public abstract void BuildModOptions();
+
+        /// <summary>
+        /// Compares one <see cref="ModOptions"/> object to another, based on their <see cref="Name"/>
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <returns></returns>
+        public int CompareTo(object obj)
+        {
+            return Name.CompareTo(obj);
```

Comment:
A `CompareTo` method that takes an `object` should do a few basic checks first, 

```CSharp
if (ReferenceEquals(this, obj) // Is the same object
    return 0;

if (obj is ModOptions other) // type safe check
    return Name.CompareTo(other.Name);
```
And then this because the catch-all case at the end
```CSharp
return Name.CompareTo(obj);

Reply:
Addressed in commit 0be7232